src/hotspot/cpu/riscv/nativeInst_riscv.cpp

```diff
@@ -84,6 +86,10 @@ bool NativeInstruction::is_load_pc_relative_at(address instr) {
          check_load_pc_relative_data_dependency(instr);
 }
 
+bool NativeInstruction::is_call_at(address addr) {
+  return NativeCall::is_at(addr);
```

Comment:
parent class calling into child class?

Reply:
We have this every where, hotspot is not an OO design, e.g.:
`bool Thread::is_JavaThread_protected(const JavaThread* target)`
Parent class have an method which takes a child as parameter.

Reply:
You're right, no, it's not for OO design.
But that's not the argument to break it easily.
Maybe in this case (`is_call_at `) it is necessary to do so? or we can find some better way? I'm not sure, I'll have further check.